config.js

```diff
@@ -60,10 +60,12 @@ const CACHE_EXPIRY_IN_SECS = pe.CACHE_EXPIRY_IN_SECS ||
   DEFAULT_CACHE_EXPIRY_IN_SECS;
 
 // request limiter settings
-const rateLimit = pe.DDOS_RATE_LIMIT;
-const rateWindow = pe.DDOS_RATE_WINDOW;
-const endpointToLimit = pe.DDOS_ENDPOINT_TO_LIMIT;
-const httpMethodToLimit = pe.DDOS_HTTP_METHOD_TO_LIMIT;
+const expressLimiterPath = pe.EXPRESS_LIMITER_PATH;
+const expressLimiterMethod = pe.EXPRESS_LIMITER_METHOD;
+const expressLimiterLookup =
+  configUtil.csvToArray(pe.EXPRESS_LIMITER_LOOKUP || 'headers.UserName');
```

Comment:
how's this? 
So the user would specify "headers.UserName,headers.x-forwarded-for"

Reply:
ok, but then we need to be more verbose in the documentation around how to format the lookup

Reply:
Does it work if a CSV is passed.? A test to prove that this work will be good.

Reply:
just updated it.

Reply:
Just modify the existing test to use a comma-separated lookup instead of just headers.UserName :)

Reply:
I was talking about the limiter. We do not have any test to show that the lookup works when it is asked to lookup by more than one parameters.

Reply:
Ohhh, no, we don't. Let's create a story to add that--don't want too much scope creep on Jon's story

Reply:
Actually, I was already working on that. Just checked it in.